api/src/main/java/org/openmrs/module/commonreports/reports/ChildCareReportManager.java

```diff
@@ -0,0 +1,467 @@
+		    muacMeasuredLessThan115);
+		
+		// adding columns
+		characteristicsDatasetDef.addColumn(aCol1, createCohortComposition(males, under6m), null);
+		characteristicsDatasetDef.addColumn(aCol2, createCohortComposition(females, under6m), null);
+		characteristicsDatasetDef.addColumn(aCol3, createCohortComposition(allGenders, under6m), null);
+		characteristicsDatasetDef.addColumn(aCol4, createCohortComposition(males, _6To23m), null);
+		characteristicsDatasetDef.addColumn(aCol5, createCohortComposition(females, _6To23m), null);
+		characteristicsDatasetDef.addColumn(aCol6, createCohortComposition(allGenders, _6To23m), null);
+		characteristicsDatasetDef.addColumn(aCol7, createCohortComposition(males, _24To59m), null);
+		characteristicsDatasetDef.addColumn(aCol8, createCohortComposition(females, _24To59m), null);
+		characteristicsDatasetDef.addColumn(aCol9, createCohortComposition(allGenders, _24To59m), null);
+		
+		// adding rows
+		characteristicsDatasetDef.addRow(MessageUtil.translate("commonreports.report.childCare.total.children.seen"),
+		    _0To60m, null);
+		characteristicsDatasetDef.addRow(
+		    MessageUtil.translate("commonreports.report.childCare.children.seen.forThe.firstTime"), childrenSeenFirstTime,
+		    parameterMappings);
+		characteristicsDatasetDef.addRow(MessageUtil.translate("commonreports.report.childCare.children.measuredFor.muac"),
+		    childrenMeasuredForMuac, parameterMappings);
+		characteristicsDatasetDef.addRow(MessageUtil.translate("commonreports.report.childCare.children.measuredFor.weight"),
+		    childrenMeasuredWeight, parameterMappings);
+		characteristicsDatasetDef.addRow(
+		    MessageUtil.translate("commonreports.report.childCare.children.with.muac.between115And125"),
+		    childrenMeasuredForMuacBetween115And125, parameterMappings);
+		characteristicsDatasetDef.addRow(MessageUtil.translate("commonreports.report.childCare.with.muac.lessThan115"),
+		    childrenMeasuredForMuacLessThan115, parameterMappings);
+		
+		return characteristicsDatasetDef;
+	}
+	
+	private CohortCrossTabDataSetDefinition createFateOfChildDatasetDefinition(AgeCohortDefinition under6m,
+	        AgeCohortDefinition _6To23m, AgeCohortDefinition _24To59m, Map<String, Object> parameterMappings) {
+		CohortCrossTabDataSetDefinition fateOfChildDatasetDef = new CohortCrossTabDataSetDefinition();
+		fateOfChildDatasetDef.addParameters(getParameters());
+		
+		Concept resultOfVisitQuestion = inizService.getConceptFromKey("report.childCare.resultOfVisitQuestion.concept");
+		
+		VisitType vt = vs.getVisitTypeByUuid(inizService.getValueFromKey("report.childCare.malnutrition.visitType.uuid"));
+		VisitCohortDefinition malnutritionChildren = new VisitCohortDefinition();
+		malnutritionChildren.setVisitTypeList(Arrays.asList(vt));
+		
+		// admitted
+		SqlCohortDefinition childrenOfFirstVisit = new SqlCohortDefinition();
+		String sql = "SELECT v.patient_id FROM visit v "
```

Comment:
Not entirely sure but it looks like you are searching for if this is the first Malnutrition visit.
However, in the report requirement it says that this "Admission" field is
> if first visit (in nutrition form first visit section filled out)

Which means: if the question `First Visit` is answered `Yes`.

<img width="485" alt="Screenshot 2021-07-28 at 22 41 05" src="https://user-images.githubusercontent.com/10692811/127392861-fe44cd94-f98a-43f7-81af-39dbdcdc6f43.png">

Reply:
@Ruhanga , I am not sure why you did not fix this.
Looks like there is a misunderstanding as to what this admission is.

I believe you have fixed the wrong query.
At this point we should call again as I think this is not clear.